chart/templates/scheduler/scheduler-deployment.yaml

```diff
@@ -264,6 +264,7 @@ spec:
           {{- if .Values.scheduler.logGroomerSidecar.args }}
           args: {{- tpl (toYaml .Values.scheduler.logGroomerSidecar.args) . | nindent 12 }}
           {{- end }}
+          envFrom: {{- include "custom_airflow_environment_from" . | default "\n  []" | indent 10 }}
```

Comment:
It feels a little weird to include the custom env if we aren't also including the standard env.

There is also #46003 that was opened to allow custom env vars just for the log groomers, which is probably a better answer.